kbs/config/docker-compose/kbs-config.toml

```diff
@@ -2,6 +2,14 @@ sockets = ["0.0.0.0:8080"]
 auth_public_key = "/opt/confidential-containers/kbs/user-keys/public.pub"
 insecure_http = true
 
+[repository_config]
+type = "LocalFs"
+dir_path = "/opt/confidential-containers/kbs/storage"
+
+[plugin_manager_config]
+work_dir = "/opt/confidential-containers/kbs/plugin"
+enabled_plugins = []
+
```

Comment:
This would have an overall workdir for all plugins. Maybe we should let every single plugin has its own configs, like the workdir of them own?

Some config format in my mind
```toml
insecure_http = true
insecure_api = true

[attestation_token_config]
attestation_token_type = "CoCo"

[grpc_config]
as_addr = "http://127.0.0.1:50004"
pool_size = 200

[repository_config]
type = "LocalFs"
dir_path = "/opt/confidential-containers/kbs/storage"

[plugins.nebula]
config_path = "/etc/kbs/plugin/nebula-config.toml"

[plugins.another]
config_a=xxx
config_b=yyy
```

If only `[plugins.nebula]` is explicitly specified in the toml, `nebula` be initialized. In this way every single plugin could have its own config, or specify the config path inside that.

This could be implemented via
```rust
#[derive(Clone, Debug, Deserialize)]
pub struct KbsConfig {
...
    pub plugins: Option<PluginManagerConfig>,
}

#[derive(Clone, Debug, Deserialize)]
struct PluginManagerConfig {
    #[cfg(feature = "nebula-plugin")]
    nebula: Option<NebulaConfig>,

    #[cfg(feature = "another-plugin")]
    another: Option<AnotherPluginConfig>,
}

#[cfg(feature = "nebula-plugin")]
#[derive(Clone, Debug, Deserialize)]
struct NebulaConfig {
    config_path: String,
}

#[cfg(feature = "nebula-plugin")]
#[derive(Clone, Debug, Deserialize)]
struct AnotherPluginConfig {
    config_a:  String,
    config_b: String,
}
```